src/webview/App.tsx

```diff
@@ -54,6 +55,7 @@ const App = () => {
           <SingleDisplay vscode={vscode} />
           <ResultsDisplay vscode={vscode} />
         </Tabs>
+        <ScoreBreakdown perc={10} oper={25} unde={20} robu={30} />
```

Comment:
score breakdown goes inside of repo display. see the figma for exactly its positioning